### DISCUSSION TITLE
How can I use g-helper with XG Mobile

### DISCUSSION THREAD

**Discussion by bypeaxel** [seerge/g-helper#3330] [Q&A] [answered]
Hello,

Can I use g-helper with xg mobile without armoury crate?

**Comment by seerge** [comment]
@wilker286 hello, G-Helper does not need Armoury Crate to work at all. You can activate and set settings for XGMobile of course.

**Reply by bypeaxel** [reply]
When I connect the XG Mobile to my computer, G-Helper doesn’t recognize it. Nothing happens when I plug it in, even when I switch the button on the cable.

**Reply by seerge** [reply, marked as answer]
@wilker286 did you try opening and closing G-Helper window?

App doesn't monitor XGM existence in realtime to preserve resources as 99% of the users don't even know what it is ... 
It checks for XGM only on start and when window is shown.

**Reply by bypeaxel** [reply]
Oh, right now its working, thank you! Do you have any tip how can I check if any app is using my XGM? Or maybe how to disable all aps that uses XGM?

**Reply by seerge** [reply]
I think nvidia control panel has an option to show all apps using GPU (including a special icon in tray)

**Reply by bypeaxel** [reply]
Last question, can I break something in my XGM if some apps are still running when I disconnect the XGM in G-Helper?

**Reply by seerge** [reply]
@wilker286 i don't think so. But apps (and especially games) using XGM may crash. So it's still better to quit everything in advance :)
